_data/languages.yml

```diff
@@ -3,3 +3,4 @@ zh_CN: 简体中文
 zh_TW: 繁體中文
 ja: 日本語
 es: Español
+tr: Turkish
```

Comment:
Türkçe?

Reply:
yep türkçe (eng: Turkish)

Reply:
Ok. the word has been replaced with its own language (5c8082c)